modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/AppearanceThemeSettingsAdapter.kt

```diff
@@ -108,11 +108,8 @@ class AppearanceThemeSettingsAdapter(
                 return
             }
             val beforeTheme = selectedTheme
-            var afterTheme = list[bindingAdapterPosition]
+            val afterTheme = list[bindingAdapterPosition]
             val validTheme = !afterTheme.isPlus || isPlusSignedIn
-            if (afterTheme.isPlus && !isPlusSignedIn) {
-                afterTheme = beforeTheme // prevent changing theme
-            }
```

Comment:
I removed this check (and the identical check in `AppearanceThemeSettingsAdapter` because:

1. These checks were added in order to make sure that a users selection appropriately updated to an allowed theme when they tried to select a plus theme but did not have a plus account (see 2062-gh-automattic/pocket-casts-android-private), but I do not see that issue occurring even with the checks removed.

3. There is code in place making it so that a user who (1) taps on a plus theme/icon, (2) upgrades to plus in the upgrade prompt that pops up, will then (3) have the plus icon/theme they tapped on activated automatically, but this was broken by these checks. So removing those checks fixed that behavior.